src/apps/components/pages/ClassPropertiesTab.svelte

```diff
@@ -332,6 +332,15 @@
             }
         }
 
+        &__field--input {
+            border: 1px solid #7a7971;
+  	    border-radius: var(--a5e-border-radius-standard);
+	    margin-bottom: 0.25rem;
+	    padding-left: 35%;
+	    padding-right: 35%;
```

Comment:
This can be changed to `padding-inline`. It's better to use `rem` as the unit instead of percentages.

Reply:
Completed.